en/docs/security-guidelines/secure-engineering-guidelines/secure-coding-guidlines/general-recommendations-for-react-secure-coding.md

```diff
@@ -2,20 +2,21 @@
 title: General Recommendations for React Secure Coding
 category: security-guidelines
 published: January 26, 2021
+version: 1.0
 ---
 
 # General Recommendations for React Secure Coding
 <p class="doc-info">Version: 1.0</p>
 ___
 
 ## Introduction
-This document summarizes the [WSO2 Secure Coding Guidelines]({{base_path}}/security-guidelines/secure-engineering-guidelines/secure-coding-guidlines/introduction/) that should be followed by WSO2 engineers while developing **React** based WSO2 products, as well as applications used within the organization. 
+This document summarizes the [WSO2 Secure Coding Guidelines](introduction.md) that should be followed by WSO2 engineers while developing **React** based WSO2 products, as well as applications used within the organization. 
 
-The purpose of this document is to increase the security awareness and make sure the products and the applications developed by WSO2 are inherently secure, by making sure security best practices are followed throughout the Software Development Life Cycle. 
+The purpose of this document is to increase security awareness and make sure the products and the applications developed by WSO2 are inherently secure, by making sure security best practices are followed throughout the Software Development Life Cycle. 
 
 
 ## Data Binding
-Untrusted user inputs must be handled by react's default data binding with curly brackets. because react’s data binding feature has capability to neutralize the html tags by default. So malicious user's input such as xss script will not be executed on the client side[^1].
+Untrusted user inputs must be handled by the default data binding of React curly brackets. As this binding feature of React can neutralize the HTML tags by default. So malicious user inputs such as the XSS script will not be executed on the client side[^1].
```

Comment:
If we change `..such as the XSS script..` to `..such as XSS scripts..`, will it be wrong?

Reply:
`such as XSS scripts` is also correct Chathura. :)